### DISCUSSION TITLE
how do I manually install the sync and submit aliases?

### DISCUSSION THREAD

**Discussion by josharian** [arxanas/git-branchless#1021] [Q&A] [answered]
I had an [existing git-related tool on my machine](https://pkg.go.dev/golang.org/x/review/git-codereview) that had already squatted the aliases sync and submit. I no longer use it. However, when I ran branchless init for the first time, it skipped setting up its own aliases. I removed the old aliases, and would like to install the branchless aliases now, but I don't know how to do that. Rerunning branchless init again doesn't do it, and I don't want to go rooting around through the source code to find out what the right answer is. Can you advise? Thanks very much.

**Comment by claytonrcarter** [comment, marked as answer]
I could have sworn that `branchless init` installed missing aliases when I've run it in the past, but maybe that was only for newly introduced aliases? 🤷 

Anyway, they are just regular git aliases, stored in the repo-local config file `.git/branchless/config`. You can either edit that file manually to add them, or run something like `git config -f .git/branchless/config alias.sync 'branchless sync'` etc. You'll have to do that in each affected repo. (You can also just install the aliases globally, w/ the risk being that you may run it in a repo that hasn't been `init`ed.)

**Reply by josharian** [reply]
Ah--that was the data point I needed. `git sync` -> `git branchless sync`, and the same for submit. Thanks!

**Reply by arxanas** [reply]
git-branchless will install missing aliases, but if you *already* have an alias of the same name installed, then git-branchless will not overwrite it so that you're not surprised. The intention is that you should manually install conflicting aliases with `git config`.

**Reply by josharian** [reply]
Yep. That all made sense to me. The problem was that after it refused to overwrite them, I didn’t know _what_ to manually set them to. Maybe the “not overwriting” message could provide the commands one would run manually as desired.
